biscuit-auth/src/datalog/mod.rs

```diff
@@ -887,7 +896,16 @@ pub fn get_schema_version(
             .iter()
             .any(|c: &Check| c.queries.iter().any(|q| !q.scopes.is_empty()));
 
-    let contains_check_all = checks.iter().any(|c: &Check| c.kind == CheckKind::All);
+    let mut contains_check_all = false;
+    let mut contains_reject_if = false;
+
+    for c in checks.iter() {
```

Comment:
Couldn't be simplified with a `contains` ?

Reply:
we are setting two separate flags, so using `contains` (or `any` as before) would require traversing the list twice.